src/netbase.cpp

```diff
@@ -165,6 +220,16 @@ bool Lookup(const char *pszName, CService& addr, int portDefault, bool fAllowLoo
     return true;
 }
 
+/**
+ * Resolve a service string with a numeric IP to its first corresponding
```

Comment:
maybe add a TODO to refactored out LookupNumeric by first-corresponding-service Lookup it seems feasible?

Reply:
please don't add TODOs to the comments, especially not for refactors